data/entities.json5

```diff
@@ -1442,6 +1494,21 @@
       'widget.intercom.io',
       'nexus-websocket-a.intercom.io',
     ],
+    products: [
+      {
+        name: 'Intercom Widget',
+        urlPatterns: [
+          '.*widget\\.intercom\\.io.*',
+          '.*js\\.intercomcdn\\.com/shim\\.latest\\.js'
+        ],
+        facades: [
+          {
```

Comment:
the amount of duplication here suggests we might have gotten our schema wrong 😆 

if we see this happen with more facades in the future we might want to extract these and have an ID for them or something

Reply:
Perhaps the shift to `entities.js` would be helpful in this space as well?

Reply:
good call I'll do the JS refactor and publish